app/app.js

```diff
@@ -201,7 +236,16 @@ function initializeConnections() {
           function: 'initializeConnections',
         });
         // Start periodic 10 second connection probe check
-        probeId = setInterval(checkConnections, 10000);
+        probeId = setInterval(checkConnections, 30000);
```

Comment:
Is there a reason that connections are checked every 30s rather than every 10s? If it needs to be changed then the comment on line 238 should also be updated.

Reply:
hmm, maybe just left over from testing! good catch, will ever to 10s

Reply:
fixed